src/utils/transclude-directive.js

```diff
@@ -0,0 +1,46 @@
+
```

Comment:
You'd just test each type of transclusion for the correct scope and DOM structure.  Should be pretty similar to other directive tests.